tools/eth-migration/fetch-control-keys.mjs

```diff
@@ -0,0 +1,98 @@
+// Only way to silence PolkadotJS API warnings we don't want
+console.warn = () => {};
+
+import { ApiPromise, WsProvider } from "@polkadot/api";
+
+const SOURCE_URL = "wss://1.rpc.frequency.xyz";
```

Comment:
is it worth making this || with an environment value?

Reply:
Sure